app/controllers/dashboard_controller.rb

```diff
@@ -344,6 +351,9 @@ def send_generated_pdf(output_path)
   def search_results(search_key)
     return nil if search_key.nil?
 
-    Collection.search(search_key).unrestricted + DocumentSet.search(search_key).unrestricted
+    collections_query = Collection.search(search_key).unrestricted.includes(:owner)
+    document_sets_query = DocumentSet.search(search_key).unrestricted.includes(:owner)
+
+    collections_query + document_sets_query
```

Comment:
Again, we access owner from search_results so preload that too.

Reply:
I like it!